test/run.d

```diff
@@ -16,7 +16,8 @@ import std.algorithm, std.conv, std.datetime, std.exception, std.file, std.forma
 import core.stdc.stdlib : exit;
 
 const scriptDir = __FILE_FULL_PATH__.dirName.buildNormalizedPath;
-string resultsDir = scriptDir.buildPath("test_results");
+auto testPath(R)(R path) { return buildNormalizedPath(scriptDir, path); }
```

Comment:
scriptDir is already normalized in the line above.

Reply:
Right, but `path` may not be normalized.